src/index.tsx

```diff
@@ -16,3 +17,5 @@ root.render(
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
 reportWebVitals();
+Panelbear.load("3TV3m1B7O7p");
```

Comment:
Better to put this directly after import, to load it as soon as possible - as it is used in your application.

Reply:
makes sense 👍🏻